.github/workflows/sync-issue-with-pr.yaml

```
@@ -0,0 +1,75 @@
+          echo "issues=$ISSUE_NUMBERS" >> $GITHUB_ENV
+
+      - name: Read Issue Labels and Titles
+        id: issue_data
+        run: |
+          PATCH_PREFIXES=("Fix:" "Infra:" "Chore:") # Add more patch-related prefixes here
+
+          for ISSUE_NUMBER in ${{ env.issues }}
+          do
+            TITLE=$(gh issue view $ISSUE_NUMBER --json title --jq '.title')
+            LABELS=$(gh issue view $ISSUE_NUMBER --json labels --jq '.labels[].name')
+          
```

yamllint

[error] trailing spaces

(trailing-spaces)